src/ui/table/additional/AdditionalInfoWidget.cpp

```diff
@@ -29,7 +29,7 @@ AdditionalInfoWidget::AdditionalInfoWidget()
     connect(m_additionalInfoLineEdit, &QLineEdit::returnPressed, this, [this] {
         m_additionalInfoLineEdit->clearFocus();
         calculateWidth();
-        emit additionalInfoEdited();
+        if (this->getMainInfoText().toStdString() != "") emit additionalInfoEdited();
```

Comment:
This might still trigger a signal emit if some information has already been entered:
1. Enter an arbitrary text and press Return
2. Click the line edit again and press Return without entering any text
-> Save emit is triggered although nothing has changed.

A better idea might be to save the line edit text every time the line edit is clicked and compare the saved text to the text after the Return key has been pressed.